yarn.lock

```diff
@@ -16354,6 +16354,24 @@ msw@^0.20.5:
     statuses "^2.0.0"
     yargs "^15.4.1"
 
+msw@^0.21.2:
+  version "0.21.2"
+  resolved "https://registry.npmjs.org/msw/-/msw-0.21.2.tgz#74ed10b8eb224325652a3c3812b5460dac297bd8"
+  integrity sha512-XOJehxtJThNFdMJdVjxDAbZ8KuC3UltOlO5nQDks0Q1yzSUqqKcVUjbKrH7T+K2hckBr0KEY2fwJHv21R4BV2A==
+  dependencies:
+    "@open-draft/until" "^1.0.3"
+    "@types/cookie" "^0.4.0"
+    chalk "^4.1.0"
+    chokidar "^3.4.2"
+    cookie "^0.4.1"
+    graphql "^15.3.0"
+    headers-utils "^1.2.0"
+    node-fetch "^2.6.1"
```

Comment:
i see msw itself uses node-fetch - but we preferred cross-fetch?

Reply:
cross-fetch uses node-fetch too, it's mostly just narrowing types and packaging other implementations